packages/maskbook/src/extension/options-page/DashboardComponents/WalletAssetsTable.tsx

```diff
@@ -70,6 +72,11 @@ export function WalletAssetsTable(props: WalletAssetsTableProps) {
     if (!chainIdValid) return null
     if (!detailedTokens.length) return null
 
+    const rankedDetailedTokens = detailedTokens.sort((a, b) => {
```

Comment:
Note: https://caniuse.com/mdn-javascript_builtins_array_sort_stable

Reply:
Plz sort tokens in the hook (`useAssetsDetailedCallback`) instead of the UI component.

Reply:
done